lua/refactoring/debug/debug_utils.lua

```diff
@@ -0,0 +1,22 @@
+local M = {}
+
+function M.get_code_gen(refactor, bufnr)
```

Comment:
move to task

```

local function require_code_gen(...)
    return function(refactor)
        ... fill me in daddy ...
        return true, refactor
    end
end

```

Reply:
Moved to task, once merged I can do for 106, etc